streams/src/main/java/org/apache/kafka/streams/kstream/internals/InternalStreamsBuilder.java

```diff
@@ -166,8 +168,8 @@ public String newStoreName(final String prefix) {
         return prefix + String.format(KTableImpl.STATE_STORE_NAME + "%010d", index.getAndIncrement());
     }
 
-    public synchronized void addStateStore(final StoreBuilder builder) {
-        internalTopologyBuilder.addStateStore(builder);
+    public synchronized void addStateStore(final StoreBuilder<KeyValueStore> builder) {
```

Comment:
It is not only possibly be a KeyValueStore right?

Reply:
ack, will update